README.md

```diff
@@ -1,16 +1,38 @@
 # Cumulocity IoT Alarm App
 
-The Cumulocity IoT Alarm App brings Alarms on your mobile phone. You'll see an overview about all Alarms raised on your Cumulocity IoT tenant, browse or filter the list of Alarms, collaborate by adding comments or modify the Alarm's severity or status.
+The Cumulocity IoT Alarm App seamlessly delivers alarms to your mobile phone, offering a comprehensive overview of all alarms raised on your Cumulocity IoT tenant. You can effortlessly browse or filter through the list of alarms, collaborate by adding comments, and modify the alarm's severity or status.
 
-Update or create an Alarm on your Cumulocity IoT tenant to trigger a push notification to be sent to your mobile phone. Open the notification to quickly see the Alarm on the Alarm App. A common use case is to inform certain users when problems occur at certain devices. 
+By utilizing this open-source application, you can significantly reduce the development time of your own application while benefiting from enhanced source code quality and reliability. The application provides a range of functionalities, including:
 
-The Cumulocity IoT Alarm App is available for Android and iOS. Open the app on your mobile phone, enter a Cumulocity IoT tenant along with your user credentials and start exploring.
+- Provides visibility of device alarms and events on the Cumulocity IoT platform. ​
+- Get details like number of alarms, first occurrences, alarm status and alarm recency.​
```

Comment:
@rahultsag  `alarm recency`. No, we're not showing the alarm's `count`.

Reply:
let's remove line 8 then.